src/config/corsConfig.ts

```diff
@@ -0,0 +1,18 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+const corsConfig = () => {
+
+  const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
For now I just added some filler code to distinguish between 'development' and 'production'. Will need to add more cases later when we push up to stage and production.